pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -74,12 +75,15 @@ func fixBundleCached() error {
 	}
 	if constants.BundleEmbedded() {
 		bundleDir := filepath.Dir(constants.DefaultBundlePath)
-		err := os.MkdirAll(bundleDir, 0775)
-		if err != nil && !os.IsExist(err) {
+		if err := os.MkdirAll(bundleDir, 0775); err != nil && !os.IsExist(err) {
```

Comment:
I think `os.IsExist()` will never be returned if `bundleDir` already exists and is a directory. So this check is unneeded.

Reply:
done